user-guide/Getting_started/The_DataMiner_User_Interface/Monitoring_app/Card_Pane/Trending_Page.md

```diff
@@ -29,4 +29,11 @@ The time span for a trend graph can be configured:
 
 ## Histograms
 
-The time span for a histogram can be configured on the left of the trending page.
+On the left, you can configure the following settings:
+
+- *Relative frequency*: Determines whether the frequency on the Y-axis should be shown in percent.
+- *Use custom intervals*: Allows you to specify a custom number of intervals and interval range.
+
+Prior to DataMiner 10.3.4/10.4.0, the time span for a histogram can also be configured on the left.
```

Comment:
The time range buttons for the histogram are actually added in 10.3.5. The buttons for the trend graph in 10.3.4 (RN35595). Apart from that, LGTM!

Reply:
Updated - thanks!